edx_credentials_themes/templates/edx.org/images/watermark.svg

```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" class="watermark-svg" viewBox="0 0 1488.09 1532.83"><defs><style>.cls-1{fill:#f2f0ef;}.cls-2{fill:#fbfaf9;}</style></defs><g id="Layer_2" data-name="Layer 2"><g id="Layer_1-2" data-name="Layer 1"><polygon class="cls-1" points="1193.71 0 1488.09 0 1161.18 1532.83 866.81 1532.83 1193.71 0"/><polygon class="cls-2" points="0 0.07 1478.28 0.07 1152.28 1532.83 0 1532.83 0 0.07"/></g></g></svg>
```

Comment:
Just a quick question -- the screenshots the "slash" portion looks a lot darker than the current version. Is this still the older version with the darker slash? Or is this the newest version? I couldn't tell easily from the preview when looking at the "rich diff" version within the PR.

Reply:
This is the newest asset, and I got the all clear from concerned parties. It does look slightly different, and I didn't bring it up explicitly. However an asset change is pretty easy for these certs.